blog/2022/12-16-resoto-aws-permissions-rollout/index.md

```diff
@@ -0,0 +1,123 @@
+
+This stack can be deployed in a single account or organization-wide as a Cloudformation StackSet.
+
+The source code for these stacks can be found in the [resoto-cf](https://github.com/someengineering/resoto-cf) Github repository.
+
+<!--truncate-->
+
+The latest versions of the stacks are hosted in the following S3 locations:
+
+- [https://resotopublic.s3.amazonaws.com/cf/resoto-role.template](https://resotopublic.s3.amazonaws.com/cf/resoto-role.template) - This stack creates the ResotoAccess role and its permissions.
+- [https://resotopublic.s3.amazonaws.com/cf/resoto-stackset.template](https://resotopublic.s3.amazonaws.com/cf/resoto-stackset.template) - This stack creates a StackSet that deploys the `ResotoAccess` role to all accounts in your organization.
+
+Using these stacks, you can easily manage the permissions required by Resoto, saving time and making it easier to maintain your AWS environment. Give them a try and see how they can help you with your AWS permissions management.
+
+## Deploying the StackSet
+
+The StackSet is the easiest way to deploy the `ResotoAccess` role to all accounts in your organization. It is also the recommended way to deploy the role to multiple accounts.
+
+This post assumes that your organization is already set up with an AWS Organization and that it meets all [the prerequisites required to use StackSets in your organization](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs.html).
+
+### Deploying the StackSet from the AWS Console
+
+#### Finding the Organization Unit (OU)
+
+Find out the Organization Unit (OU) in your organization where you want to deploy the `ResotoAccess` role. The role will be deployed to all accounts in this OU and all child OUs.
+
+Navigate to the Organization Home in the [AWS Organizations Console](https://console.aws.amazon.com/organizations/v2/home/accounts) and find the OU where you want to deploy the role.
+
+![Organization Home](./img/find_the_ou.png)
+
+The OU is that cryptic string underneat the name of the OU. In our example we want to deploy the role to all accounts in the organization, so we will use the Root OU. Which in this case is `r-7h7x`.
+
+#### Deploying the StackSet
+
+If you are not already in the correct account, log into the account where you want to deploy the StackSet. This account will be the account that manages the StackSet and the role that is deployed to all accounts in the organization unit.
+
+Navigate to the [Cloudformation Console](https://console.aws.amazon.com/cloudformation/home) and click on `Create Stack`. Leave all the default settings, enter the before mentioned S3 URL `https://resotopublic.s3.amazonaws.com/cf/resoto-stackset.template` in the `Amazon S3 URL` field and click `Next`.
```

Comment:
You can create a deep link like this: [Deploy Template](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?templateURL=https://resotopublic.s3.amazonaws.com/cf/resoto-stackset.template&stackName=ResotoAccess)